templates/consul_catalog.tmpl

```diff
@@ -34,6 +34,27 @@
     interval = "{{ $healthCheck.Interval }}"
   {{end}}
 
+  {{ $buffering := getBuffering $service.Attributes }}
+  {{if $buffering }}
+  [backends.backend-{{ $backendName }}.buffering]
+    enabled = {{ $buffering.Enabled }}
+    {{if $buffering.MaxRequestBodyBytes }}
```

Comment:
You don't have to check if each attribute is present.
You can just check `buffering` in *all* the templates.